pkg/cmd/cli/cmd/idle_test.go

```diff
@@ -0,0 +1,316 @@
+package cmd
```

Comment:
This needs test/cmd tests.  You should create synthetic endpoints and verify idle works on them.